test/functional/terminal/tui_spec.lua

```diff
@@ -1384,3 +1384,12 @@ describe('TUI background color', function()
     wait_for_bg('light')
   end)
 end)
+
+it("waits for background terminal response (hitting timeout)", function()
+  command = nvim_prog .. ' -u NONE --cmd q'
+  local start = os.time()
```

Comment:
Could use luv.hrtime() for higher granularity

Reply:
Tried that first (from anohter test), but it did not appear to work here.

Reply:
The test has been removed for now.. if re-added it needs a more fine-grained timer then, because it is not taking 1s anymore.